src/index.ts

```diff
@@ -9,7 +11,8 @@ export default class TabContainerElement extends HTMLElement {
       if (!(target instanceof HTMLElement)) return
       if (target.getAttribute('role') !== 'tab' && !target.closest('[role="tablist"]')) return
       const tabs = Array.from(this.querySelectorAll('[role="tablist"] [role="tab"]'))
-      const currentIndex = tabs.indexOf(tabs.find(tab => tab.matches('[aria-selected="true"]')))
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Looks like linter passes with a warning without this. Is this rule new? We use this a lot in dotcom so might want to just allow it.

Reply:
I think this is a rule that already exists. I just looked at dotcom and it seems that it's because we don't extend `plugin:github/typescript` in dotcom 😬